BroadcomCloudSwg/connectors/broadcom_cloud_swg_connector.py

```diff
@@ -72,18 +76,43 @@ def last_event_date(self) -> datetime:
             return last_event_date
 
     @classmethod
-    def rate_limiter(cls) -> AsyncLimiter:  # pragma: no cover
+    def rate_limiter(cls) -> AsyncLimiter | None:  # pragma: no cover
         """
         Get or initialize rate limiter.
 
         Returns:
             AsyncLimiter:
         """
         if cls._rate_limiter is None:
-            cls._rate_limiter = AsyncLimiter(1, 1)
+            requests_limit = os.getenv("REQUESTS_PER_SECOND_TO_INTAKE")
+            if requests_limit is not None and int(requests_limit) > 0:
+                cls._rate_limiter = AsyncLimiter(int(requests_limit), 1)
+
+            if requests_limit is None:
+                cls._rate_limiter = AsyncLimiter(1, 1)
```

Comment:
Can you add a condition when `requests_limit` is `0` in order to deactivate the ratelimiter?

Reply:
OK. My bad. I misunderstood the code